Library/Formula/antlr.rb

```diff
@@ -7,10 +5,33 @@ class Antlr < Formula
 
   def install
     prefix.install "antlr-#{version}-complete.jar"
-    bin.write_jar_script prefix/"antlr-#{version}-complete.jar", "antlr4"
+    bin.write_jar_script jar, "antlr4"
     (bin+"grun").write <<-EOS.undent
       #!/bin/bash
       java -classpath #{prefix}/antlr-#{version}-complete.jar:. org.antlr.v4.runtime.misc.TestRig "$@"
     EOS
   end
+
+  def jar
+    prefix/"antlr-#{version}-complete.jar"
+  end
```

Comment:
It's OK, just inline this in both places, function makes it a bit harder to follow the e.g. install method.